testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/SocialLoginTest.java

```diff
@@ -255,6 +257,16 @@ public void openshift4Login() {
         testTokenExchange();
     }
 
+    @Test
+    public void openshift4KubeAdminLogin() {
+        setTestProvider(OPENSHIFT4_ADMIN_LOGIN);
+        ((OpenShiftLoginPage) this.currentSocialLoginPage).setUserLoginLinkTitle(getConfig(currentTestProvider, "loginBtnTitle"));
```

Comment:
I was thinking about this and I think we should move this line directly to `setTestProvider` (of course with some `if` to limit this to OpenShift4 providers). This concerns all OpenShift4 tests, not just kubeAdmin.

Reply:
@vmuzikar True, that'd be a better place for this. 

Another point: This value needs only to be set when the OCP login page has some boxes to click, leading to the actual login form, which is not the case for every OCP environment, is it? So the `if` should only be used in the case of this provider, not for e.g. `OPENSHIFT4`. Right? 
Apart from that: I think renaming the Provider enum to OPENSHIFT4_BOXED_LOGIN (other naming suggestions are appreciated) makes more sense, too, because it's possible to not only get to the login for the admin user, but depending on `loginBtnTitle` for other users, too. What do you think?

Reply:
@vmuzikar @DGuhr Can you please share the Docker Image that I can use for testing. I could not download the zip file which @DGuhr shared with me over the Gchat.

Reply:
@vmuzikar I added the proposed changes

Reply:
@vmuzikar again added the proposed changes and rebased. Please review if you find the time :) 
@iam-veeramalla I'll send you an updated docker image which you can download from quay.